go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/shermp/Kobo-UNCaGED
 
 require (
+	github.com/bamiaux/rez v0.0.0-20170731184118-29f4463c688b
 	github.com/disintegration/imaging v1.6.0
```

Comment:
You need to run `go mod tidy` to clean up the unused dependencies.

Reply:
Ah, oops. Thanks for catching that